reporter/src/main/kotlin/reporters/TableReporter.kt

```diff
@@ -47,6 +47,11 @@ abstract class TableReporter : Reporter() {
              */
             val vcsInfo: VcsInfo,
 
+            /**
+             * A list containing all evaluator errors. `null` if no evaluator result is available.
```

Comment:
Below you're setting `evaluatorErrors` to an empty list if there is no evaluator result. So should the type by non-nullable?

Reply:
No, it should be null, forgot to change that.